### DISCUSSION TITLE
Menüstruktur Gerätekataloge

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#22] [Idea retired / denied]
Aktuell gibt es unter Geräte einen Menüpunkt "Gerätekataloge" (Mehrzahl?) und unter Meine Einrichtung "Eigene Gerätearten" und "Eigene Gerätetypen".

![Bildschirmfoto 2022-08-10 um 14 27 12](https://user-images.githubusercontent.com/80519295/183902974-e369a5d4-679c-4ce9-afdc-4322ff519d68.png)

Könnte man diese nicht zusammenfassen und gemeinsam unter Geräte > Gerätekataloge abbilden?

Design Vorschlag:

![Bildschirmfoto 2022-08-10 um 14 37 44](https://user-images.githubusercontent.com/80519295/183909567-2bcbcaf7-d2d4-4594-979a-3716b161c3e8.png)

![Bildschirmfoto 2022-08-10 um 15 10 03](https://user-images.githubusercontent.com/80519295/183911450-b35d2dfa-934f-4072-b446-48a56668d894.png)

![Bildschirmfoto 2022-08-10 um 15 31 32](https://user-images.githubusercontent.com/80519295/183916287-50c087d4-0a34-4632-89a3-7fa5d5963033.png)

Optimierungen Eigene Gerätearten:

![Bildschirmfoto 2022-08-10 um 15 23 22](https://user-images.githubusercontent.com/80519295/183912902-ba59b6e6-08b8-4561-961e-3bd9d0135d98.png)

![Bildschirmfoto 2022-08-10 um 15 27 08](https://user-images.githubusercontent.com/80519295/183913536-28363487-abbe-4cb3-aad8-599f91f683c1.png)

**Comment by YvesR** [comment]
Obsolete due MDM implementation > Tags > DeviceTypes and DeviceModels re-designed.
